combinetf2/tfhelpers.py

```diff
@@ -29,3 +31,42 @@ def is_diag(x):
     return tf.math.equal(
         tf.math.count_nonzero(x), tf.math.count_nonzero(tf.linalg.diag_part(x))
     )
+
+
+def is_on_gpu(tensor):
```

Comment:
Is it maybe better to use `tf.config.list_physical_devices('GPU')` instead? Avoids the need of passing a tesnor. It should also be used with care as neither of the two works in tf.function.

Reply:
We want to leave the possibility to run on CPU even if a GPU is present, so in this case we would have to either refer back to the config, or do it like this by checking the tensor.